complete/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Is this file needed? Because the same entry is in the top-level .gitignore.

Reply:
Removed.